### PR TITLE
add comments regarding 2.4.1 alpha revisions

### PR DESCRIPTION
**Pull request by DG12**
# 01/07/19 2.4.1 added flash, nrf_nordic_flash,     sdk_overrides/ & (SDK_ROOT)/components/ble/ble_radio_notification

### CODE REVIEW

**Comment by DG12** [comment]
Enabling LOG causes loader errors trying to resolve nRF52_log_frontend_std_xxx functions. :-(

File: ruuvi_examples/ruuvi_firmware/sdk_application_config.h [hunk 1]
```diff
@@ -20,7 +22,10 @@
   #define CRC16_ENABLED   FDS_CRC_ENABLED  
   #define CRC32_ENABLED   FDS_CRC_ENABLED  
 #endif
-#define NRF_LOG_ENABLED 0  // Disable log printout by default to save space
+
+#ifndef NRF_LOG_ENABLED
```
Comment (ojousima) [inline ruuvi_examples/ruuvi_firmware/sdk_application_config.h]: Checking for if NRF_LOG_ENABLED is enabled here might cause some hard to track down bugs. It's better to throw error on redefinition. Please remove

Comment (DG12) [reply ruuvi_examples/ruuvi_firmware/sdk_application_config.h]: I don't understand what you are saying.

Without the` ifndef ` here and only the `define NRF_LOG_ENABLED 0`
main.c cannot `#define NRF_LOG_ENABLED 1`

Please reconsider.

Comment (ojousima) [reply ruuvi_examples/ruuvi_firmware/sdk_application_config.h]: Exactly, If some .c files have log enabled and some do not there might be issues with conditional compiling of Nordic SDK log module.

Comment (DG12) [reply ruuvi_examples/ruuvi_firmware/sdk_application_config.h]: I have removed the conditional setting of NRF_LOG_ENABLED as you requested.

**Comment by DG12** [comment]
I think I now understand some of your earlier comments like, "too much output" from debug log.
++++++++++++++++++++++++++++++++++++
In version 2.2 from 07/11/18 I successfully used NRF_LOG_ENABLED only in main.c with  NRF_LOG_DEFAULT_LEVEL 4. This cause all the log out put FROM MAIN ONLY. (actually now that I look carefully at it is seems there are some entries from other initializations too). 
See http://mybeacons.info/log.2.2.4.html
++++++++++++++++++++++++++++++++++++
Refer to http://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.sdk5.v13.0.0%2Fnrf_log.html&cp=4_0_0_3_20

> Configuration
> ...
> A module can have a section that overrides default logger settings for the given module. Modules can override the following configuration:
> 
> Enabling logs in the module.
> Independent lowest severity level.
> ...

++++++++++++
What caused you to add "NRF_LOG_ENABLED"  to sdk_application_config.h ?
Did you encounter a problem?

**Comment by ojousima** [comment]
> What caused you to add "NRF_LOG_ENABLED" to sdk_application_config.h ?
Did you encounter a problem?

The definition should be same in every .c file, and enabling / disabling logging should be a per-file macro. There's a lot of SDK examples for per-file definitions.

**Pull request merged**